### PR TITLE
[MRG] Update documentation with links to new SBTs and a tutorial

### PR DESCRIPTION
**Pull request by ctb**
Should be merged after #183 and #175.

Various documentation updates, including:

* Add links to new SBTs made by @luizirber.
* Add tutorial from https://github.com/ngs-docs/2017-lapaz-assembly.
* Include and improve #193 (better TOC).

Note that the this depends on changes in #183 and #175, and so the `pip install` needs to be updated after those are merged; and the URLs for the `eschericia-sigs.tar.gz` and `shakya-unaligned-contigs.sig` files used in the tutorial should be updated to use `master` before merge.

- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make coverage` Is the new code covered?
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by ctb** [comment]
Ready for review!

File: doc/tutorials.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+pip install -U pip
+pip install -U Cython
+pip install -U jupyter jupyter_client ipython pandas matplotlib scipy scikit-learn
+
+pip install https://github.com/dib-lab/khmer/archive/master.zip
+pip install https://github.com/dib-lab/sourmash/archive/spacegraphcats.zip
+
+```
+
+## Fingerprint reads
```
Comment (luizirber) [inline doc/tutorials.md]: We should stick to one nomenclature: signature or fingerprint?

File: doc/tutorials.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+Download some reads and a reference genome:
+
+```
+mkdir ~/data
+cd ~/data
+wget http://public.ged.msu.edu.s3.amazonaws.com/ecoli_ref-5m-trim.pe.fq.gz
+wget https://s3.amazonaws.com/public.ged.msu.edu/ecoliMG1655.fa.gz
+```
+
+Compute a scaled MinHash fingerprint from our reads:
```
Comment (luizirber) [inline doc/tutorials.md]: same as above

File: doc/tutorials.md (hunk 1 above)
Comment (ctb) [inline doc/tutorials.md]: fixed in 7bdb1df

**Pull request merged**